cmake/WarpXFunctions.cmake

```diff
@@ -345,10 +351,7 @@ function(warpx_print_summary)
     message("        bin: ${CMAKE_INSTALL_BINDIR}")
     message("        lib: ${CMAKE_INSTALL_LIBDIR}")
     message("    include: ${CMAKE_INSTALL_INCLUDEDIR}")
-    message("      cmake: ${CMAKE_INSTALL_CMAKEDIR}")
-    if(WarpX_HAVE_PYTHON)
-        message("     python: ${CMAKE_INSTALL_PYTHONDIR}")
-    endif()
```

Comment:
Just double checking: these lines with `CMAKE_INSTALL_PYTHONDIR` (and not `CMAKE_INSTALL_CMAKEDIR`) were also removed on purpose?

Reply:
yes, as per 2nd commit they were unused